src/FormBuilder/resources/js/form-builder/src/middleware/bangBang.ts

```diff
@@ -0,0 +1,15 @@
+export default (blocks) => {
```

Comment:
```suggestion
import type {BlockInstance} from '@wordpress/blocks';

export default (blocks: BlockInstance[]) => {
```

Reply:
This file will be deleted. It was a test file when first setting up the middleware.